app/views/casework/crime_applications/_interests_of_justice.html.erb

```diff
@@ -27,17 +27,26 @@
       <% end %>
     </tbody>
   </table>
+<% elsif crime_application.ioj_passport.include?('on_age_under18') %>
+  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= label_text(:justification) %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <%= t('on_age_under18', scope: 'values') %>
+      </dd>
+    </div>
+  </dl>
 <% else %>
   <dl class="govuk-summary-list govuk-!-margin-bottom-9">
-    <% crime_application.ioj_passport.each do |passport| %>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          <%= label_text(:justification) %>
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= t(passport, scope: 'values') %>
-        </dd>
-      </div>
-    <% end %>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= label_text(:justification) %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <%= t('on_offence', scope: 'values') %>
```

Comment:
Given what happened with the means_passport, I wonder if it would be foolhardy to have this as a default else if there is no ioj_reasons, and the ioj_passport does not contain the age passport. 

@timpeat I'd appreciate your thoughts. Is there a sensible default else clause to show them something went wrong as a fallback for if Apply lets something through that it shouldn't

Reply:
I'd be inclined to do something like:

``` ruby
      <dd class="govuk-summary-list__value">
        <%= t(crime_application.ioj_passport.sort.join('_and_'), scope: 'values') %>
        </dd>
```
and then add a translation for key:
``` yaml
    on_age_under18_and_on_offence: Not needed based on offence and because the client is under 18 years old
    on_age_under18: Not needed because the client is under 18 years old
    on_offence: Not needed based on offence

Reply:
> So when it is passported on under 18 that trumps the passport on offence, so they want that to be the only one that shows.

In which case you could just change "on_age_under18_and_on_offence:" value in locale file to "Not needed because the client is under 18 years old"